src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -433,8 +433,12 @@ namespace NServiceBus
     }
     public class FailedConfig
     {
+        [System.ObsoleteAttribute("Use `FailedConfig(string errorQueue, HashSet<Type> unrecoverableExceptionTypes)` " +
+            "instead. Will be removed in version 7.0.0.", true)]
```

Comment:
You need "treat as error from X" as well since true == error == breaking change?